plugins/ssr/ssrcommon.ml

```diff
@@ -906,7 +906,8 @@ let pf_interp_ty ?(resolve_typeclasses=false) ist gl ty =
      | _ -> (mkCCast ty (mkCType None)).v)) ty in
      mk_term ' ' (force_type ty) in
    let strip_cast (sigma, t) =
-     let rec aux t = match EConstr.kind_of_type sigma t with
+    let open EConstr in
+     let rec aux t = match kind_of_type sigma t with
```

Comment:
I don't get these patches, the old code was using `EConstr.kind_of_type`, now it uses `open EConstr` (but continues to use `EConstr.isSort` and co). I think you should just revert this part of the patch, it is not needed anymore (it used to be needed I guess when the code was moved to ssrcommon).

Reply:
Because warning 40 is enabled, the constructors of the type also need to be in scope for the patterns. (They used to be in Constr which is globally opened in this file)

Reply:
Well spotted. Then please remove the others `EConstr.` in the scope of `open EConstr`.

Reply:
I am not super keen on touching lines that are not directly related to the PR. Does it hurt to overqualify?

Reply:
Can I push a commit doing it?

Reply:
I mean, the argument of minimizing a diff for reviewing is valid, but if the resulting code is crap I prefer suffering a bit more in the review and end up with decent code (anyway, I'm the reviewer for this file).

Reply:
Fair enough, please do it if you're faster than me.